src/applications/claims-status/tests/components/OverviewPage.unit.spec.jsx

```diff
@@ -14,63 +15,23 @@ const params = { id: 1 };
 describe('<OverviewPage>', () => {
   const store = createStore(() => ({}));
 
-  context('DDL feature flag is enabled', () => {
-    const claim = {
-      attributes: {
-        open: false,
-        decisionLetterSent: true,
-      },
-    };
-
-    it('should render a link to the claim letters page when using Lighthouse', () => {
-      const screen = render(
-        <Provider store={store}>
-          <OverviewPage
-            claim={claim}
-            useLighthouse
-            showClaimLettersLink
-            params={params}
-            clearNotification={() => {}}
-          />
-        </Provider>,
-      );
-
-      screen.getByText('Get your claim letters');
-    });
-
-    it('should render a link to the claim letters page when using EVSS', () => {
-      const screen = render(
-        <Provider store={store}>
-          <OverviewPage
-            claim={claim}
-            showClaimLettersLink
-            params={params}
-            clearNotification={() => {}}
-          />
-        </Provider>,
-      );
-
-      screen.getByText('Get your claim letters');
-    });
-  });
-
   context('when claim is closed', () => {
     const claim = {
```

Comment:
Reordering some of the keys to show the be ordered the same as they are when Lighthouse sends them to us. Also updated a few places to make the values more realistic to what occurs in production